### PR TITLE
catinabox unit tests, pycon2016 tutorial

### PR DESCRIPTION
**Pull request by ndcampbell**
Pycon2016 tutorial :)

### CODE REVIEW

File: tests/test_safecatmath.py [hunk 1]
```diff
@@ -19,17 +19,24 @@ def test__cat_years_to_hooman_years__0__returns_0():
 
 
 def test__cat_years_to_hooman_years__less_0__raises():
+    with pytest.raises(safecatmath.InvalidAge):
+        safecatmath.cat_years_to_hooman_years(-1)
     assert True
```
Comment (tcervo) [inline tests/test_safecatmath.py]: Do you need assert True when using pytest.raises?

Comment (ndcampbell) [inline tests/test_safecatmath.py]: Just tried it, you are right, that is unnecessary. Thanks!

**Comment by steve98654** [comment]
Hey ndcampbell, 

When we implement the test__cat_years_to_hooman_years__middle_age__succeeds test, 
should we just check if the function evaluates without error, or actually returns the correct value? 

For example, you wrote: 

 def test__cat_years_to_hooman_years__middle_age__succeeds():
-    assert True
-    assert catmath.cat_years_to_hooman_years(7) == 35

which checks that the function takes 7 as input and returns 35 as output where as 

 def test__cat_years_to_hooman_years__middle_age__succeeds():
     assert catmath.cat_years_to_hooman_years(7) 

which just check if the function fails.  If someone changed the cat years to human years from anything other than 5, then the first test above would fail, but the second would not.  Which one do you think is the more appropriate test?

**Comment by ndcampbell** [comment]
@steve98654 

I think the first test is more appropriate since it would be testing the actual return value against the input. In the next part of the tutorial it looks like they create tests for validating if the function runs or throws the correct exception. I think a test for both is useful but in this case they are looking for return value validation

**Pull request closed without merging**